router/router.go

```diff
@@ -1120,9 +1120,19 @@ func (worker *workerT) postStatusOnResponseQ(respStatusCode int, respBody string
 
 		worker.rt.failedEventsChan <- *status
 
-		if respStatusCode >= 500 {
-			timeElapsed := time.Since(firstAttemptedAtTime)
-			if respStatusCode != types.RouterTimedOutStatusCode && respStatusCode != types.RouterUnMarshalErrorCode {
+		switch respStatusCode {
+		case transformer.DropStatusCode:
+			status.JobState = jobsdb.Aborted.State
```

Comment:
shouldn't we skip adding the userID to both failed & abort map?

Reply:
agreed

Reply:
I am not seeing the `addToFailedMap = false` thoug

Reply:
it's done further down
https://github.com/rudderlabs/rudder-server/pull/2230/files/6a09470fed0e412a3a38561b82896afcb1843c4d#diff-c399d26e7f23b713ce01946ac9e0e6f1fc94d692e8a086180191901da91b6964R1152